conf/systemd.service

```diff
@@ -8,38 +8,30 @@ User=__APP__
 Group=__APP__
 WorkingDirectory=__INSTALL_DIR__/
 ExecStart=__INSTALL_DIR__/mautrix-discord -c __INSTALL_DIR__/config.yaml
-StandardOutput=append:/var/log/__APP__/__APP__.log
-StandardError=inherit
 Restart=always
 RestartSec=3
 
-# Sandboxing options to harden security
-# Depending on specificities of your service/app, you may need to tweak these
-# .. but this should be a good baseline
-# Details for these options: https://www.freedesktop.org/software/systemd/man/systemd.exec.html
-ReadWritePaths=__INSTALL_DIR__ /var/log/__APP__
+# Optional hardening to improve security
+ReadWritePaths=__INSTALL_DIR__/ /var/log/__APP__
 NoNewPrivileges=yes
 MemoryDenyWriteExecute=true
-PrivateTmp=yes
 PrivateDevices=yes
-PrivateUsers=true
-RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
-RestrictNamespaces=yes
-RestrictRealtime=yes
+PrivateTmp=yes
+ProtectHome=yes
+ProtectSystem=strict
+ProtectControlGroups=true
 RestrictSUIDSGID=true
-DevicePolicy=closed
-ProtectClock=yes
-ProtectHostname=yes
-ProtectProc=invisible
-ProtectSystem=full
-ProtectControlGroups=yes
-ProtectKernelModules=yes
-ProtectKernelTunables=yes
```

Comment:
Unless you are sure it is not required (for example if these options are covered by other hardening directives), I propose to keep these:

 - `RestrictNamespaces=yes`
 - `RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK`
 - `ProtectProc=invisible`
 - `DevicePolicy=closed`

My point of view is that we may keep these unless we find that removing them are required for the process to work correctly. I know that it may make debugging harder when we face an issue.

I am open to discussion about this though.

Reply:
I've try with this and it's ok ! 
-   RestrictNamespaces=yes
-   RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
-   ProtectProc=invisible
-   DevicePolicy=closed

I can add it 👍